Source/inv.cpp

```diff
@@ -483,15 +483,15 @@ void DrawInv(CelOutputBuffer out)
 					    InvRect[j + SLOTXY_INV_FIRST].X + RIGHT_PANEL_X,
 					    InvRect[j + SLOTXY_INV_FIRST].Y - 1,
 					    pCursCels, frame, frame_width,
-						false);

```

Comment:
please avoid causing differences unrelated to the PR, a clang format commit can come later

Reply:
Yes..  I did this before figuring out that I can apply clang-format only to the changes of the PR. Applying to the whole file send to move more than intended.

Reply:
Yes..  I did this before figuring out that I can apply clang-format only to the changes of the PR. Applying to the whole file send to move more than intended.

Reply:
We are working on solving that problem today :)